### PR TITLE
WSO2 Font integration and Custom components.

### PR DESCRIPTION
**Pull request by menakaj**

### CODE REVIEW

File: components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/Application/Edit/Base/baseLayout.css [hunk 1]
```diff
@@ -60,8 +52,7 @@
 /*Tab styling*/
 div.tab {
     float: left;
-    border: 1px solid #ccc;
```
Comment (sinthuja) [inline components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/Application/Edit/Base/baseLayout.css]: The css classes files should go into the public directory right? Else we can't change the theming..

File: components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/UIComponents/FloatingButton/FloatingButton.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ */
+
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import './floatingButton.css';
+
+class FloatingButton extends Component {
+
+    handleClick(event) {
+        console.log("click");
```
Comment (sinthuja) [inline components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/UIComponents/FloatingButton/FloatingButton.jsx]: Need to remove all console.log. Please do have a todo cleanup all these.

File: components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/index.css [hunk 3]
```diff
@@ -48,54 +48,48 @@ body {
...
     background-color: #ffffff;
+    padding: 0;
 }
 
 #header-content {
-    height: 100px;
+    height: 125px;
     width: 100%;
     margin: 0 10px 0 0;
-    background-color: #BDBDBD;
```
Comment (sinthuja) [inline components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/index.css]: Again, we need to move all css to public folder.

**Review by sinthuja — CHANGES_REQUESTED** [review]
We shouldn't have the css files within the main source, rather we should have it in the public directory. else we can't have theming support.

**Comment by menakaj** [comment]
All the internal css were extracted and moved to external css file.

**Comment by menakaj** [comment]
TODOs were added to remove console logs. 
Method names were changed to meaningful names.
Cleaned up the code base.

**Review by sinthuja — APPROVED** [review]

**Pull request merged**